### PR TITLE
Grammar corrections in community folder

### PR DESCRIPTION
**Pull request by DamianArado**
Grammar corrections in 20 files in community folder with 196 additions and 204 deletions.

### CODE REVIEW

File: community/committee.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
               <h1>C++ Committee Meeting FAQ for Boost Members</h1>
 
               <p><strong>Who can attend C++ Committee meetings?</strong>
-              Members of PL22.16 (the INCITS/ANSI committee) or of a WG21 (ISO)
+              Members of PL22.16 (the INCITS/ANSI committee) or of WG21 (ISO)
```
Comment (mloskot) [inline community/committee.html:36]: The `a` is incorrect in this context. The WG21 is one an known.

File: community/counted_body.html [hunk 2]
```diff
@@ -79,14 +79,13 @@ <h2>And then there were none...</h2>
                 offer a transparent optimisation. The body is shared between
                 copies of a string until such a time as a change is needed,
                 at which point a copy is made. Such a COPY ON WRITE pattern
-                (a specialisation of LAZY EVALUATION) requires the use of a
+                (a specialization of LAZY EVALUATION) requires the use of a
```
Comment (mloskot) [inline community/counted_body.html:82]: Well, I'd argue this change is incorrect  ;P

File: community/counted_body.html [hunk 3]
```diff
@@ -135,31 +134,31 @@ <h2>Attached vs detached</h2>
...
               Library), now part of the C++ standard library. The STL
-              approach makes extensive use of compile time polymorphism based
-              on well defined operational requirements for types. For
+              approach makes extensive use of compile-time polymorphism based
+              on well-defined operational requirements for types. For
               instance, each container, contained and iterator type is
               defined by the operations that should be performable on an
               object of that type, often with annotations describing
-              additional constraints. Compile time polymorphism, as its name
+              additional constraints. Compile-time polymorphism, as its name
```
Comment (mloskot) [inline community/counted_body.html:157]: Please, revert this part

File: community/counted_body.html [hunk 4]
```diff
@@ -241,7 +240,7 @@ <h2>A requirements based approach</h2>
               </table>
 
               <p>Note that the two arguments to <code>dispose</code> are to
-              support selection of the appropriate type safe version of the
+              support selection of the appropriate type of safe version of the
```
Comment (mloskot) [inline community/counted_body.html:243]: Rather `type-safe`

File: community/error_handling.html [hunk 5]
```diff
@@ -68,11 +68,11 @@ <h3>When should I use exceptions?</h3>
               example, a desktop application performing a long computation
               might periodically check to see whether the user had pressed a
               cancel button. Throwing an exception could allow the operation
-              to be cancelled gracefully. On the other hand, it would
+              to be canceled gracefully. On the other hand, it would
```
Comment (mloskot) [inline community/error_handling.html:71]: Thin ice!

File: community/error_handling.html [hunk 6]
```diff
@@ -161,7 +161,7 @@ <h3>How should I design my exception classes?</h3>
                 the error</strong> in your exception class' public interface.
                 A fixation on the <code>what()</code> message is likely to
                 mean that you neglect to expose information someone might
-                need in order to make a coherent message for users. For
+                need to make a coherent message for users. For
```
Comment (mloskot) [inline community/error_handling.html:164]: Hmm?

File: community/generic_programming.html [hunk 7]
```diff
@@ -461,7 +461,7 @@ <h2><a name="policy" id="policy">Policy Classes</a></h2>
                 elementary components.</p>
               </blockquote>
 
-              <p>Andrei's description of policy classes suggests that their
+              <p>Andrei's description of policy classes suggests that there
```
Comment (mloskot) [inline community/generic_programming.html:464]: Wrong!

File: community/gsoc.html [hunk 8]
```diff
@@ -30,27 +30,26 @@ <h1>Boost C++ Libraries</h1>
...
-              for funding the coming summer. Students should expect to be
+              for funding the upcoming summer. Students are expected to be
               flexible with regard to what topics and proposals mentors are
               willing to mentor.</p>
               
-              <p>If you arrive to the process too late to prearrange a mentor,
+              <p>If you arrive at the process too late to prearrange a mentor,
               some potential mentors supply prewritten project proposals with
-			  C++ programming competency tests on <a href="https://github.com/boostorg/wiki/wiki/Google-Summer-of-Code%3A-Overview">
+        C++ programming competency tests on <a href="https://github.com/boostorg/wiki/wiki/Google-Summer-of-Code%3A-Overview">
```
Comment (mloskot) [inline community/gsoc.html:50]: Still needs reindentation

File: community/gsoc.html [hunk 9]
```diff
@@ -68,70 +67,70 @@ <h3 id="thisyear">Google Summer of Code 2020 overview</h3>
...
 <ul>
-	<li>Vikram Singh Chundawat will work in taking Real to revision ready state. His focus will be in improving the internal representation datatypes 
-		and providing functions required to conveniently support Taylor series.
-	<i>Mentors</i>: Damian Vicino and Laouen Belloli
-	</li>
+  <li>Vikram Singh Chundawat will work in taking Real to revision ready state. His focus will be in improving the internal representation datatypes 
+    and providing functions required to conveniently support Taylor series.
+  <i>Mentors</i>: Damian Vicino and Laouen Belloli
+  </li>
 </ul>
```
Comment (mloskot) [inline community/gsoc.html:134]: Please, do not mix fixes and reformatting in the same PR

File: community/gsoc_2006_boost_overview.html [hunk 10]
```diff
@@ -716,7 +715,7 @@ <h3><a name="scope_of_projects" id=
                 <li>utilities and tool projects using Boost.</li>
               </ul>
 
-              <p>Of these, additions (like for instance the mincut/maxflow
+              <p>Of these, additions (like for instance the min-cut/maxflow
```
Comment (mloskot) [inline community/gsoc_2006_boost_overview.html:718]: Rather "max-flow min-cut algorithm"

File: community/review_schedule.html [hunk 11]
```diff
@@ -27,19 +27,19 @@
...
-		recent reviews already completed; that helps track review
-		manager assignments and libraries reviewed but not yet posted
-		on the web site. There is often a lag between acceptance and
-		site posting as authors address issues raised in the formal
-		review.</p>
+    recent reviews already completed; that helps track review
+    manager assignments and libraries reviewed but not yet posted
+    on the website. There is often a lag between acceptance and
+    site posting as authors address issues raised in the formal
+    review.</p>
```
Comment (mloskot) [inline community/review_schedule.html:42]: Reformatting does not belong here

**Review by mloskot — CHANGES_REQUESTED** [review]

**Review by mloskot — APPROVED** [review]

File: community/feature_model_diagrams.html [hunk 12]
```diff
@@ -47,10 +47,10 @@ <h2>Introduction</h2>
               discussions. The Whorf hypothesis that "Language shapes the way
               we think, and determines what we can think about" seems to
               apply. In discussion of library variability issues, we have
-              been crippled by lack of a good language. With feature models
+              been crippled by lack of a good language. With feature models,
               we now have a language to carry on the dialog.</p>
 
-              <p>The graphical feature diagrams presented by C&amp;E are not
+              <p>The graphical feature diagrams presented by C&amp; E are not
```
Comment (glenfe) [inline community/feature_model_diagrams.html:53]: C&E was right, not "C& E"

File: community/feature_model_diagrams.html [hunk 13]
```diff
@@ -120,7 +120,7 @@ <h2><a id="FeatureDescriptions" name=
 
               <h2>What is a Feature?</h2>
 
-              <p>A feature [C&amp;E 4.9.1] is "anything users or client
+              <p>A feature [C&amp; E 4.9.1] is "anything users or client
```
Comment (glenfe) [inline community/feature_model_diagrams.html:123]: Again, no space here.

File: community/implementation_variations.html [hunk 14]
```diff
@@ -41,7 +41,7 @@ <h2>Separation of interface and implementation</h2>
               portability, and standardization, the interface is what is
               important, while the implementation is just a detail.</p>
 
-              <p>Dietmar K&uuml;hl, one of the original boost.org
+              <p>Dietmar K&uuml; hl, one of the original boost.org
```
Comment (glenfe) [inline community/implementation_variations.html:44]: No space here.

File: community/official_library_maintainer_program.html [hunk 15]
```diff
@@ -39,8 +39,8 @@ <h2>Introduction</h2>
               Boost Community Maintenance Team</a> (CMT) has identified many Boost 
               libraries still used but not maintained actively. To protect the
               reputation that Boost has cultivated over the years, as an
-              organization that provides high quality open source software,
-              it is critical that all libraries bundled with Boost releases
+              organization that provides high-quality open-source software,
+              all libraries must bundle with Boost releases
```
Comment (glenfe) [inline community/official_library_maintainer_program.html:43]: This is wrong. The original "bundled" is what we want here.

**Review by glenfe — CHANGES_REQUESTED** [review]

**Comment by DamianArado** [comment]
Grammar corrections in 19 files in the community folder with 153 additions and 161 deletions.

**Pull request merged**